doc/README-ENCDataVault.md

```diff
@@ -0,0 +1,20 @@
+Cracking ENCSecurity DataVault
+==============================
+
+1. Run encdatavault2john.py on vault folder.
+
+```bash
+$ ../run/encdatavault2john.py path/to/vault > hash
+```
+
+2. Run john on the file.
+
+```bash
+$ ../run/john hash
+```
+
+The decryption test is done on a 32-bit magic value, meaning that occasional false positives can occur. Cracked passwords for those false positives will not decrypt the vault correctly.
+
+The plugin have beem tested on Sony ENCDataVault lite 6.2.13, SanDisk SecureAccess 3.02 and ENC DataVault 7.1.1W.
```

Comment:
Typo: `beem`. And we generally use the word "format" in the documentation.

Also, lines longer than 79, which we generally avoid in documentation (except for lengthy command-lines, URLs, etc.), although in an `.md` file it's arguably more acceptable.